src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/BuildingRequestResolver.java

```diff
@@ -87,14 +87,13 @@ public boolean canResolve(
         return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1168)